core/src/main/java/com/oath/oak/Chunk.java

```diff
@@ -174,10 +173,10 @@ private boolean casEntriesArray(int item, int offset, int expected, int value) {
     /**
      * write key in slice
      **/
-    void writeKey(K key, int ei) {
+    private void writeKey(K key, int ei) {
```

Comment:
I would leave this package private, since this is sort of an internal API (like `readKey`, etc.)